src/ComScoreStreamingAnalytics.ts

```diff
@@ -154,9 +173,9 @@ export class ComScoreStreamingAnalytics {
   private transitionToAd(): void {
     if (this.comScoreState !== ComScoreState.Advertisement) {
       this.stopComScoreTracking();
-      this.streamingAnalytics.playVideoAdvertisement({
-        ns_st_cl: Math.round(this.currentAd.duration),
-      }, this.adType());
+      var metadata: any = { ns_st_cl: Math.round(this.currentAd.duration)};
```

Comment:
Seems like this can be a `const`
```suggestion
      const metadata: any = { ns_st_cl: Math.round(this.currentAd.duration) };
```

Reply:
fixed